### PR TITLE
V3 overview/clean code

### PR DESCRIPTION
**Pull request by charliestock**
This pull request updates the overview section for generic_COBALT.F90 to better reflect the current COBALTv3 configuration.  Changes are limited to commenting at the beginning of the code.  There should be no answer changes.

### CODE REVIEW

File: generic_tracers/generic_COBALT.F90 [hunk 1]
```diff
@@ -1,53 +1,52 @@
...
-!              for consistency.
-!           5) The default carbon chemistry routine is now MOCSY
-!           6) The iron scavenging has been re-tuned to new atmospheric (Ginoux-AM4),
-!              sediment, river and hydrothermal vent (Tagliabue) sources.
-!
-!       parameterizations used herein, as well as definitions for all variables
-!       and parameters.  The 33 model state variables are:
+!  COBALTv3 (Stock et al., submitted) was developed to enhance the robustness of COBALT
+!  across open ocean and coastal regions.  It now divides the phytoplankton community
+!  into 3 rather than 2 size classes to better resolve phytoplankton commmunities from
```
Comment (yichengt900) [inline generic_tracers/generic_COBALT.F90]: commmunities -> communities.

Should we also consider rephrasing this sentence to reflect that COBALTv3 includes small, medium, and large phytoplankton, as well as a diazotroph?

**Comment by yichengt900** [comment]
@charliestock , thanks for updating the overview section. It looks good, I only have a small comment and a suggestion.

File: generic_tracers/generic_COBALT.F90 (hunk 1 above)
Comment (charliestock) [reply generic_tracers/generic_COBALT.F90]: Thanks for checking this Yi-Cheng.  How about we add the sentence: "Thus, COBALTv3 now includes small, medium, and large phytoplankton, as well as a diazotroph." after the Van Oostende reference.  The existing sentence clarifies the rationale behind the change, while the second ensures that the implications for the state variables are clear.

What is the quickest way/best process to modify a pull request?

Comment (yichengt900) [reply generic_tracers/generic_COBALT.F90]: @charliestock, sounds good. In your case, you can simply use the GitHub UI to edit the file and save the change (see the screenshot below):

edit

Comment (charliestock) [reply generic_tracers/generic_COBALT.F90]: Thanks for the pointer.  I think I managed to implement the suggested changes successfully, and I captured one additional typo.  Can you see the changes?

File: generic_tracers/generic_COBALT.F90 [hunk 2]
```diff
@@ -1,53 +1,52 @@
...
+!  plankton foodwebs.
 !
-!       Version 2.0 has a number of refinements:
+!  COBALTv2 served as the ocean biogeochemical component of GFDL's CMIP6 earth
+!  system model (Stock et al., 2020).  The structure of COBALTv2 was largely
+!  analogous to COBALTv1, but notable adjustments included: 1) the addition of
+!  temperature and O2-dependent particle remineralization following Laufkotter
+!  et al. (2017); 2) retuning of ammonia dynamics and the addition air-sea ammonia
+!  exchanges (Paulot et al., 2020); 3) adjustments to iron sources and
+!  scavenging; and 4) movement of the defaul carbon chemistry to MOCSY.
```
Comment (kakearney) [inline generic_tracers/generic_COBALT.F90]: Typo: defaul -> default

Comment (charliestock) [reply generic_tracers/generic_COBALT.F90]: Thanks Kelly, got it!

**Review by yichengt900 — APPROVED** [review]
Thanks, @charliestock and @kakearney! The changes look good to me. The overview provides a brief history of COBALT, our main scope, and outlines the new features in COBALTv3, along with references. Approved.

**Comment by charliestock** [comment]
Thank you @yichengt900 @kakearney, have a great weekend!

**Pull request merged**